modules/oe_theme_helper/src/Plugin/Field/FieldFormatter/SocialMediaIconsListFormatter.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
If this solution is acceptable test has to be provided. See modules/oe_theme_helper/tests/src/Kernel/Plugin/Field/FieldFormatter/SocialMediaLinksFormatterTest.php.

Reply:
Test has been provided - SocialMediaIconsListFormatterTest.php